packages/web/src/pages/profile-page/components/SocialLink.tsx

```diff
@@ -135,19 +112,41 @@ const SocialLink = ({
     }
   }
 
-  return (
-    <div className={styles.socialLink}>
-      <div
-        onClick={isSingleLink ? onIconClick : () => {}}
-        className={cn(styles.wrapper, {
-          [styles.singleLink]: isSingleLink
-        })}
-      >
-        {icon}
-        {!iconOnly && <div className={styles.text}>{text}</div>}
-      </div>
+  let href: Maybe<string>
+
+  if (isHandleType(type)) {
+    href = `${SITE_URL_MAP[type]}${link}`
+  } else if (isSingleLink) {
+    if (!/^https?/.test(link)) {
+      href = `http://${link}`
```

Comment:
should this be https?

Reply:
i am copying this from existing code, but agree lets make this https